### PR TITLE
Feat/dv 135/create page to confirm goods delivery for driver

### PR DESCRIPTION
**Pull request by Eugene-pas**

### CODE REVIEW

File: src/assets/scss/assets/pages/_confirmTripOfferPage.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    margin-bottom: 20px;
+  }
+
+  @media only screen and (max-width: $max_tablet_size){
+    .confirmTripOfferBody {
+      margin: 20px;
+    }
+  }
+
+}
```
Comment (Grasmus) [inline src/assets/scss/assets/pages/_confirmTripOfferPage.scss]: Add empty line in the eof

File: src/components/confirmGoodsDeliveryForDriver/index.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                <h1 className="titel">Active Trip</h1>
+
+                {tripInfo != null ?
+                    <div className="confirmTripOfferBody">
+                        <TripInfo tripInfo={tripInfo}/>
+                        <h2>Offers</h2>
+                        <div>
+                            {offers != null ?
+                                offers.map((item, iter) => {
+                                    console.log(iter, item);
```
Comment (Grasmus) [inline src/components/confirmGoodsDeliveryForDriver/index.jsx]: Remove console log

File: src/components/confirmGoodsDeliveryForDriver/infoTrip/index.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                            <div className="leftSide">
+                                <div className="dataField">
+                                    <div className="dates">
+                                        <Tooltip
+                                            placement="top"
+                                            title={tooltipMessages.DEPARTURE_DATE}
+                                        >
+                                            <div className="date">
+                                                <FiCalendar size={DEFAULT_ICON_SIZE}/>
+                                                <p>{moment(props.tripInfo.departureDate).format('LL HH:mm') + " "}</p>
```
Comment (Grasmus) [inline src/components/confirmGoodsDeliveryForDriver/infoTrip/index.jsx]: Please, use UTC date

File: src/assets/scss/assets/pages/_confirmInfoTrip.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  }
+
+  .dataField {
+    color: #656565;
+    display: flex;
+
+    svg{
+      margin-right: 10px;
+    }
+  }
```
Comment (marynakernychna) [inline src/assets/scss/assets/pages/_confirmInfoTrip.scss]: Align code

File: src/assets/scss/assets/pages/_confirmOffers.scss [hunk 5]
```diff
@@ -0,0 +1,130 @@
+@import '../variables/colors';
+@import '../variables/others';
+
+.confirmOffers {
+
```
Comment (marynakernychna) [inline src/assets/scss/assets/pages/_confirmOffers.scss]: Remove the blank line

File: src/assets/scss/assets/pages/_confirmTripOfferPage.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    box-shadow: rgba(14, 30, 37, 0.12) 0px 2px 4px 0px,
+    rgba(14, 30, 37, 0.32) 0px 2px 16px 0px;
+    border-radius: 10px;
+    margin: 20px 100px 10px 100px;
+    padding-left: 40px;
+    padding-right: 40px;
+    padding-bottom: 10px;
+  }
+
+  .titel{
```
Comment (marynakernychna) [inline src/assets/scss/assets/pages/_confirmTripOfferPage.scss]: titel?

File: src/components/confirmGoodsDeliveryForDriver/offers/index.jsx [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+            <div>
+                <Button onClick={async () => {
+                    await confirmGoodsTransfer(
+                        {
+                            offerId: props.offerData.id,
+                            tripRole: offerRoles.DRIVER,
+                            isConfirmed: true
+                        }
+                    );
+                    console.log("Confirm", {offerId: props.offerData.id, Role: offerRoles.DRIVER, isConfirmed: true});
```
Comment (Grasmus) [inline src/components/confirmGoodsDeliveryForDriver/offers/index.jsx]: Remove console log

File: src/components/confirmGoodsDeliveryForDriver/offers/index.jsx [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+                }} className="confirm button">Confirm</Button>
+                <Button onClick={async () => {
+                    await confirmGoodsTransfer(
+                        {
+                            offerId: props.offerData.id,
+                            tripRole: offerRoles.DRIVER,
+                            isConfirmed: false
+                        }
+                    );
+                    console.log("Decline", {offerId: props.offerData.id, Role: offerRoles.DRIVER, isConfirmed: false});
```
Comment (Grasmus) [inline src/components/confirmGoodsDeliveryForDriver/offers/index.jsx]: Remove console log

File: src/assets/scss/assets/pages/_confirmTripOfferPage.scss (hunk 1 above)
Comment (marynakernychna) [inline src/assets/scss/assets/pages/_confirmTripOfferPage.scss]: Add the blank line

File: src/assets/scss/main.scss [hunk 9]
```diff
@@ -14,6 +14,9 @@
 @import "./assets/pages/offerPointTableDnD";
 @import "./assets/pages/tripInfo";
 @import "./assets/pages/driversInvites";
+@import "./assets/pages/confirmInfoTrip";
+@import "./assets/pages/confirmTripOfferPage";
+@import "./assets/pages/confirmOffers";
```
Comment (marynakernychna) [inline src/assets/scss/main.scss:19]: Import component styles to page files

File: src/components/confirmGoodsDeliveryForDriver/index.jsx [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+    }, [])
+
+    console.log(offers);
+
+    return (
+        <div>
+            <Header/>
+
+            <div className="confirmGoodsDeliveryForDriverPage">
+                <h1 className="titel">Active Trip</h1>
```
Comment (marynakernychna) [inline src/components/confirmGoodsDeliveryForDriver/index.jsx]: titel?

File: src/components/confirmGoodsDeliveryForDriver/index.jsx [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+        async function fetchData() {
+            const data = await getUserTripInfo()
+            setTripInfo(data);
+            setOffers(await getDriverConfirmGoodsDelivery(data.id));
+        }
+
+        fetchData();
+    }, [])
+
+    console.log(offers);
```
Comment (Grasmus) [inline src/components/confirmGoodsDeliveryForDriver/index.jsx]: Remove console log

**Review by marynakernychna — COMMENTED** [review]
Remove all console logs

File: src/constants/api/urls.js [hunk 12]
```diff
@@ -41,15 +41,17 @@ export const OFFERS_URLS = {
     GET_BY_USER: OFFERS_URL,
     GET_TO_CONFIRM: OFFERS_URL + "/to-confirm",
     CONFIRM_GOODS_TRANSFER: OFFERS_URL + "/confirm-goods-transfer",
-    GET_BY_NEAR_ROUT: OFFERS_URL + "/near-route"
+    GET_BY_NEAR_ROUT: OFFERS_URL + "/near-route",
```
Comment (marynakernychna) [inline src/constants/api/urls.js]: GET_BY_NEAR_ROUTE

**Review by marynakernychna — CHANGES_REQUESTED** [review]
Starts with an error

**Review by Grasmus — APPROVED** [review]

**Review by marynakernychna — APPROVED** [review]

**Pull request merged**